pkg/kudoctl/util/vars/vars.go

```diff
@@ -11,9 +11,7 @@ var (
 	Namespace            string
 	Parameter            []string
 	PackageVersion       string
-	RepoName             string
-	RepoPath             string
-	RepoURL              string
 	StorageBucket        string
 	StoragePrefix        string
+	RepoPath 			 string = "$HOME/.kudo/repository" // this won't work on windows
```

Comment:
we should abstract all this in its own logic so we can deal with it cleaner, for now it is super fine leaving it that way.